opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -220,6 +226,24 @@ def on_end(self, span: ReadableSpan) -> None:
             with self.condition:
                 self.condition.notify()
 
+    def _at_fork_reinit(self):
+        # could be in an inconsistent state after fork, reinitialise by calling `_at_fork_reinit`
+        # (creates a new lock internally https://github.com/python/cpython/blob/main/Python/thread_pthread.h#L727)
+        # if exists, otherwise create a new one.
+        if hasattr(self.condition, "_at_fork_reinit"):
```

Comment:
I'm not very clear on what this does. How can `_at_fork_reinit()` be called from `self._condition`?